lib/search.js

```diff
@@ -4,10 +4,11 @@ var cheerio = require('cheerio');
 
 var h = require('./helpers');
 
+var optsGlobal = null;
```

Comment:
there are two `doSearch` calls in this module - was not able to find easy way to pass throttle opt to second one without big changes, so left global (actually its global only for this module env)

why we should pass it into query? no need to send `throttle` to google api :)